pyproject.toml

```diff
@@ -72,6 +72,7 @@ tldextract = ">=5.1.0"
 typer = ">=0.12.0"
 typing-extensions = ">=4.1.0"
 yarl = ">=1.18.0"
+cachetools = "^5.5.1"
```

Comment:
Could you please use ">=" version specifier? And please put it in alphabetical order 🙂.